backend/link-shortener/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:16
+
+COPY . .
+WORKDIR /app
+
+RUN ["npm","install","global","@nestjs/cli"]
+RUN ["npm", "install", "redis@4.1.0" ]
```

Comment:
Redis will be installed in the next line. Not necessary here.

Reply:
Okay.